chain/network/src/peer.rs

```diff
@@ -492,15 +492,17 @@ impl StreamHandler<Vec<u8>, io::Error> for Peer {
 
                 // Receive invalid routed message from peer.
                 if !routed_message.verify() {
-                    actix::spawn(
-                        self.peer_manager_addr
-                            .send(NetworkRequests::BanPeer {
-                                peer_id: self.node_info.id.clone(),
-                                ban_reason: ReasonForBan::InvalidSignature,
-                            })
-                            .map_err(|e| error!(target: "client", "{}", e))
-                            .map(|_| ()),
-                    );
+                    if let Some(peer_info) = self.peer_info.as_ref() {
+                        actix::spawn(
+                            self.peer_manager_addr
+                                .send(NetworkRequests::BanPeer {
```

Comment:
you don't really need to `actix::spawn` if you not expecting the answer.
you can just use `let _ = self.peer_manager_addr.do_send(...)`

Reply:
done!